moveit_ros/planning_interface/test/CMakeLists.txt

```diff
@@ -1,15 +1,16 @@
-if (CATKIN_ENABLE_TESTING)
+if (BUILD_TESTING)
   find_package(moveit_resources REQUIRED)
-  find_package(rostest REQUIRED)
+  find_package(ament_cmake_gtest REQUIRED)
 
-  add_executable(test_cleanup test_cleanup.cpp)
-  target_link_libraries(test_cleanup moveit_move_group_interface)
+  # add_executable(test_cleanup test_cleanup.cpp)
+  # target_link_libraries(test_cleanup moveit_move_group_interface)
 
-  add_rostest_gtest(moveit_cpp_test moveit_cpp_test.test moveit_cpp_test.cpp)
-  target_link_libraries(moveit_cpp_test moveit_cpp ${catkin_LIBRARIES})
+  ament_add_gtest(moveit_cpp_test moveit_cpp_test.test moveit_cpp_test.cpp)
+  #ament_target_dependencies(moveit_cpp_test rclcpp)
+  target_link_libraries(moveit_cpp_test moveit_cpp)
```

Comment:
moveit_cpp isn't defined in this cmake so shouldn't it be listed under ament_target_dependencies instead?

Reply:
moveit_cpp is a library that is defined in the same package (this CMakeLists.txt is just included by the parent one). So linking the library here is fine. However, the test is not enabled yet since runtime tests only work with an initialized planning pipeline, making OMPL a dependency.

Reply:
Neat!